corehq/apps/geospatial/reports.py

```diff
@@ -59,12 +61,14 @@ class BaseCaseMapReport(ProjectReport, CaseListMixin, XpathCaseSearchFilterMixin
     def template_context(self):
         # Whatever is specified here can be accessed through initial_page_data
         context = super(BaseCaseMapReport, self).template_context
+        celery_task_tracker = get_celery_task_tracker(self.domain, base_key=INDEX_ES_TASK_HELPER_BASE_KEY)
         context.update({
             'mapbox_access_token': settings.MAPBOX_ACCESS_TOKEN,
             'saved_polygons': [
                 {'id': p.id, 'name': p.name, 'geo_json': p.geo_json}
                 for p in GeoPolygon.objects.filter(domain=self.domain).all()
             ],
+            'es_indexing_message': celery_task_tracker.get_message()
```

Comment:
nit: `task_state` or `task_status` might be better name than using service name that is powering it.

Reply:
This has been reworked according to [this](https://github.com/dimagi/commcare-hq/pull/35126#discussion_r1762796406) comment ([6906d82](https://github.com/dimagi/commcare-hq/pull/35126/commits/6906d828c78293671103f8edadef77d568694369)).